packages/react-examples/src/react/ColorPicker/ColorPicker.Basic.Example.tsx

```diff
@@ -44,11 +44,11 @@ export const ColorPickerBasicExample: React.FunctionComponent = () => {
         // If your app is localized, you MUST provide the `strings` prop with localized strings.
         strings={{
           // By default, the sliders will use the text field labels as their aria labels.
-          // If you'd like to provide more detailed instructions, you can use these props.
```

Comment:
Looks like the longer strings were added in #11430 in response to #11415, apparently with the rationale that the slider doesn't implement all default slider-like behavior and therefore more instructions are needed? (I'm not sure whether this is true.)

For ColorPicker specifically, it will automatically reuse the visible label as the aria-label if an override isn't provided, so it's not necessary to demonstrate providing the overrides if you're certain that it's almost always bad practice. (Though if you decide to remove the `strings` prop, it would be good to keep the "If your app is localized..." comment.)

It might also be good explain a little more about why providing detailed instructions is a bad practice, either here (if you keep the overrides) or maybe in the docs for these props.

Reply:
ah, good point I'll remove all but hue (which doesn't have a default), and add some stuff to the type doc